src/OperatorFactory.cpp

```diff
@@ -216,7 +219,103 @@ OperatorFactory::OperatorFactory(const ssize_t Lt,
   std::cout << "\tMeson operators initialised" << std::endl;
 }
 
-void OperatorFactory::build_vdaggerv(const std::string &filename, const int config) {
+OperatorFactory::OperatorFactory(const ssize_t Lt,
```

Comment:
This doubling of the constructor makes no sense, simply call the original constructor with ```dilE = 0```....